.gitignore

```diff
@@ -9,3 +9,7 @@ yarn-error.log
 **/.DS_Store
 /config/dev
 /tmp
+
+# editor
+.devcontainer
+.vscode
```

Comment:
We would like to keep the VS Code workspace settings, hence `.vscode` has not been added to `.gitignore`.